packages/synapse-react-client/src/lib/containers/auth/LoginPage.tsx

```diff
@@ -122,17 +139,34 @@ export default function LoginPage(props: LoginPageProps) {
                 color: '#1e4964',
               }}
             >
-              {step === 'VERIFICATION_CODE' ? (
+              {step === 'VERIFICATION_CODE' && (
                 <>
                   <Typography variant="headline1" sx={{ mb: 4 }}>
                     Enter your verification code
                   </Typography>
-                  <Typography variant="headline2">
+                  <Typography variant={'headline2'} sx={{ lineHeight: '30px' }}>
                     Enter the 6-digit, time-based verification code provided by
                     your authenticator app.
                   </Typography>
                 </>
-              ) : (
+              )}
+              {step === 'RECOVERY_CODE' && (
+                <>
+                  <Typography variant="headline1" sx={{ mb: 4 }}>
+                    Enter your backup code
+                  </Typography>
+                  <Typography variant={'headline2'} sx={{ lineHeight: '30px' }}>
+                    Your backup code is a 16-digit code. We generated 10 of
+                    these codes for you when you set up 2FA. If you don’t have
+                    access to these codes, please{' '}
+                    <Link href={LOST_ACCOUNT_ACCESS_CONTACT_URL}>
+                      contact us
+                    </Link>
+                    .
```

Comment:
TODO: This text matches the above, so I should create a reusable constant or component fragment